src/index.scss

```diff
@@ -1,5 +1,6 @@
 body {
   margin: 0;
+  padding: 0;
```

Comment:
```suggestion
```

We can remove `margin: 0` since it's already applied by Bootstrap: https://github.com/twbs/bootstrap/blob/master/scss/_reboot.scss#L41

`padding: 0` is unnecessary since I don't think browsers add any padding to the body